pkgs/by-name/mu/mum/package.nix

```
@@ -0,0 +1,64 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  cmake,
+  pkg-config,
+  alsa-lib,
+  gdk-pixbuf,
+  glib,
+  libnotify,
+  libopus,
+  openssl,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mum";
+  version = "0.5.1";
+  src = fetchFromGitHub {
+    owner = "mum-rs";
+    repo = pname;
+    rev = "v${version}";
```

```suggestion
    repo = "mum";
    tag = "v${version}";
```

use of pname like this is an antipattern and breaks overrides.